modules/custom/az_demo/az_demo.info.yml

```diff
@@ -4,6 +4,7 @@ description: 'WARNING:Do not enable this on live/production sites, it will alter
 core_version_requirement: ^9
 package: 'The University of Arizona - Experimental'
 dependencies:
+  - az_custom_menu_block
```

Comment:
```suggestion
  - az_menu_block
```

Reply:
`az_custom_menu_block` is the machine name we agreed on during the Wednesday workshop (to distinguish these from `menu_block` module blocks).

Reply:
Oh, but I see this is referencing the module name.  Never mind.